src/blocks/TitleParagraphBlock.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import { MaxWidthContainerComponent } from '../components/blockHelpers'
+import {
+  BaseBlockProps,
+  MarkdownHtmlComponent,
+  NativeColorPickerComponent,
+} from './BaseBlockProps'
+
+const SectionComponent = styled('section')(
+  ({ backgroundColor }: { backgroundColor: string }) => ({
+    width: '100%',
+    background: backgroundColor,
+    paddingTop: '120px',
+    paddingBottom: '120px',
+  }),
+)
+
+const ContentComponent = styled('div')(
+  ({ textPosition }: { textPosition: string }) => ({
+    display: 'flex',
+    flexDirection: textPosition === 'center' ? 'column' : 'row',
+    justifyContent: textPosition === 'left' ? 'space-between' : 'center',
+    alignItems: 'center',
+    textAlign: textPosition === 'center' ? 'center' : 'left',
+    width: '100%',
+  }),
+)
+
+const TitleComponent = styled('h2')(
+  ({
+    titleColor,
+    textPosition,
+  }: {
+    titleColor: string
+    textPosition: string
+  }) => ({
+    color: titleColor,
+    fontSize: '48px',
+    marginRight: textPosition === 'left' ? 'auto' : 0,
+    marginLeft: textPosition === 'right' ? 'auto' : 0,
+    width: '100%',
+    maxWidth: '50%',
+  }),
+)
+
+const ParagraphComponent = styled('div')(
+  ({
+    paragraphColor,
+    textPosition,
+  }: {
+    paragraphColor: string
+    textPosition: string
+  }) => ({
+    display: textPosition === 'right' ? 'none' : 'block',
+    marginTop: textPosition === 'center' ? '25px' : 0,
+    maxWidth: textPosition === 'left' ? '40%' : '100%',
+    width: '100%',
+    color: paragraphColor,
+    fontSize: '20px',
```

Comment:
is it possible to avoid hard-coded font sizes? It feels weird to me that one specific paragraph should be `4px` larger than the rest of the paragraphs. Haven't looked at the design though but that's a gut feel

Reply:
I agree! Folled the design as is, but there should definitely be some common styles for things such as fonts, spacings etc. Can add it in another PR.